modules/openy_features/openy_taxonomy/modules/openy_txnm_color/config/install/core.entity_form_display.taxonomy_term.color.default.yml

```diff
@@ -22,11 +21,12 @@ content:
     third_party_settings: {  }
     region: content
   field_color:
-    weight: 31
+    weight: 101
     settings:
-      color: FFFFFF
+      size: 60
+      placeholder: ''
     third_party_settings: {  }
-    type: jquery_colorpicker
+    type: string_textfield
```

Comment:
TODO: revert 8.x-2.x file before merging. This is 9.x temporary workaround

Reply:
https://github.com/ymcatwincities/openy/pull/2175/commits/c39f358c87796ef1e05fdda8cbf680523e4d0d74